internal/kafka/internal/services/quota/ams_quota_service.go

```diff
@@ -306,3 +306,37 @@ func (q amsQuotaService) DeleteQuota(subscriptionId string) *errors.ServiceError
 	}
 	return nil
 }
+
+func (q amsQuotaService) IsSubscriptionActive(kafka *dbapi.KafkaRequest) (*bool, error) {
+	billingModel, err := q.kafkaConfig.GetBillingModelByID(kafka.InstanceType, kafka.ActualKafkaBillingModel)
+	if err != nil {
+		return nil, err
+	}
+
+	orgID, err := q.amsClient.GetOrganisationIdFromExternalId(kafka.OrganisationId)
+	if err != nil {
+		return nil, err
+	}
+
+	quotas, err := q.amsClient.GetQuotaCostsForProduct(orgID, billingModel.AMSResource, billingModel.AMSProduct)
```

Comment:
can we call the variable quotaCosts?
The Quota concept is different than the QuotaCost concept in AMS